artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/cursor/impl/PageCounterRebuildManager.java

```diff
@@ -216,6 +219,10 @@ public void rebuild() throws Exception {
          try (LinkedListIterator<PagedMessage> iter = msgs.iterator()) {
             while (iter.hasNext()) {
                PagedMessage msg = iter.next();
+               if (storedLargeMessages != null && msg.getMessage().isLargeMessage()) {
+                  logger.debug("removing storedLargeMessage {}", msg.getMessage().getMessageID());
```

Comment:
As getMessageID returns a long it will be autoboxing, may be worth a gate since all the similar logging before and after this uses them already.

Reply:
I didn't think this one was a big deal.. it's only happening on startup.. but I will add the gate.